scripts/au/gnaf.sh

```diff
@@ -31,10 +35,11 @@ BOUNDARY_DIR="$(find $TMP -type d | grep -v 'Administrative Boundaries' | head -
 # load data into tables
 python /usr/local/gnaf-loader/load-gnaf.py \
     --pguser gnafun --pgdb gnafdb --pgpassword gnafpw \
-    --no-boundary-tag \
+    --gnaf-schema gnaf \
```

Comment:
the `openaddresses` table created later pulls from the gnaf schema, however by default gnaf-loader uses gnaf_monthYEAR schema. It's simpler to set it to a fixed schema here then update the `openaddresses` table query each new release.